### DISCUSSION TITLE
wg-easy/wg-easy and WeeJeWel/wg-easy

### DISCUSSION THREAD

**Discussion by mitrios-hub** [wg-easy/wg-easy#711] [Q&A] [answered]
Please explain what difference between these two projects. Thnx.

**Comment by pheiduck** [comment, marked as answer]
Since @WeeJeWel  was able to win @peterlewis  and me as new maintainers https://github.com/wg-easy/wg-easy/discussions/553 the project was spared into an organization. So the `wg-easy/wg-easy` is the more updated one. :)
I hope this answers your question.

**Reply by Iman-Sh** [reply]
Don't know whether this is the right place for asking😅, but may i ask what are the new features you guys have planned to add to wg-easy? is there gonna be time limit for peers? saving peers traffic usage?
